### ISSUE TITLE
Cannot work with the sqlite databases containing R-Tree tables

### ISSUE THREAD

**Issue by zczc2390** [duckdb/sqlite_scanner#50]
**What happens?**

When trying to list tables in a SQLite database which contains a R-Tree table. It will have errors such as "no such module: rtree" or "no such module: rtree-i32"

**To Reproduce**
1. Create a sqlite database based on the example in https://www.sqlite.org/rtree.html
- Create a sql script file with below scripts, and name it as `demo.sql`
```
CREATE VIRTUAL TABLE demo_index USING rtree(
   id,              -- Integer primary key
   minX, maxX,      -- Minimum and maximum X coordinate
   minY, maxY       -- Minimum and maximum Y coordinate
);
INSERT INTO demo_index VALUES
  (28215, -80.781227, -80.604706, 35.208813, 35.297367),
  (28216, -80.957283, -80.840599, 35.235920, 35.367825);
```
- Run sqlite3 with above script to create a database with R-Tree Table
```
sqlite3 demo.sqlite < demo.sql
```

2. Open the sqlite database created in step 1
```
duckdb ./demo.sqlite
```
3. Show all tables in the sqlite database
```
v0.8.0 e8e4cea5ec
Enter ".help" for usage hints.
D show tables;
Error: Invalid Error: Failed to prepare query "PRAGMA table_info('demo_index')": no such module: rtree
D 
```

**OS:**

Windows 10, x64, Ubuntu 20.04 LTS

**SQLite Version:**
3.31.1

**DuckDB Version:**
v0.8.0

**Comment by Mause** [comment]
This error is simply caused by the fact that we don't compile the r-tree module into the copy of sqlite3 included in the extension. You can probably compile your own copy of the extension if you absolutely need it.

**Comment by Mytherin** [comment]
Thanks for the report! This should be fixed with #70
